data/json/items/armor/arms_armor.json

```diff
@@ -934,22 +1069,123 @@
+    "description": "A pair of neoprene forearm sleeves covered with molded plastic sheaths.",
+    "weight": "175 g",
     "volume": "1250 ml",
-    "copy-from": "vambrace_larmor",
+    "price": 1750,
+    "price_postapoc": 500,
+    "to_hit": 1,
+    "material": [ "plastic", "neoprene" ],
+    "symbol": "[",
+    "looks_like": "armguard_hard",
+    "warmth": 20,
+    "material_thickness": 3,
+    "environmental_protection": 1,
+    "flags": [ "OUTER", "WATER_FRIENDLY", "BLOCK_WHILE_WORN" ],
+    "armor": [
+      {
+        "covers": [ "arm_l", "arm_r" ],
+        "encumbrance": 3,
+        "coverage": 90,
+        "cover_melee": 90,
+        "cover_ranged": 95,
+        "cover_vitals": 90,
+        "specifically_covers": [ "arm_lower_r", "arm_lower_l" ]
+      }
+    ]
+  },
+  {
+    "id": "xl_vambrace_hard",
+    "type": "ARMOR",
+    "copy-from": "vambrace_hard",
+    "looks_like": "vambrace_hard",
+    "name": { "str": "pair of XL hard vambraces", "str_pl": "pairs of XL hard vambraces" },
+    "description": "A pair of neoprene forearm sleeves covered with molded plastic sheaths.  Epic sized.",
+    "weight": "550 g",
+    "volume": "4 L",
     "extend": { "flags": [ "OVERSIZE" ] }
   },
   {
-    "id": "xs_vambrace_larmor",
+    "id": "xs_vambrace_hard",
     "type": "ARMOR",
-    "copy-from": "vambrace_larmor",
-    "looks_like": "vambrace_larmor",
-    "name": { "str": "pair of XS leather vambraces", "str_pl": "pairs of XS leather vambraces" },
-    "description": "A pair of light leather arm guards, made for archery.",
+    "copy-from": "vambrace_hard",
+    "looks_like": "vambrace_hard",
+    "name": { "str": "pair of XS hard vambraces", "str_pl": "pairs of XS hard vambraces" },
+    "description": "A pair of neoprene forearm sleeves covered with molded plastic sheaths.  Tiny sized.",
+    "proportional": { "weight": 0.75, "volume": 0.75, "price": 0.5 },
+    "extend": { "flags": [ "UNDERSIZE" ] }
+  },
+  {
+    "id": "vambrace_metal",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "pair of metal vambraces", "str_pl": "pairs of metal vambraces" },
+    "description": "A pair of vambraces hammered out from metal.  Perfect for the Post-Apocalyptic Warrior look.",
+    "weight": "500 g",
+    "volume": "1500 ml",
+    "price": 15000,
+    "price_postapoc": 500,
+    "to_hit": 1,
+    "material": [ "iron" ],
+    "symbol": "[",
+    "looks_like": "armguard_metal",
+    "color": "light_gray",
+    "warmth": 20,
+    "material_thickness": 3,
+    "environmental_protection": 1,
+    "flags": [ "OUTER", "BLOCK_WHILE_WORN", "WATER_FRIENDLY" ],
+    "armor": [
+      {
+        "covers": [ "arm_l", "arm_r" ],
+        "encumbrance": 6,
+        "coverage": 95,
+        "cover_melee": 95,
+        "cover_ranged": 90,
+        "cover_vitals": 70,
+        "specifically_covers": [ "arm_lower_r", "arm_lower_l", "arm_upper_r", "arm_upper_l" ]
```

Comment:
This contradicts the language you are trying to standardize

Reply:
Yeah, I forgot to change the coverage after copying from the metal arm guard